### PR TITLE
Component

### PR DESCRIPTION
**Pull request by mosidrum**
IN this PR I did the following:

Deleted all the boilerplate from Create React App 
Created a new directory called components.
Created a new Calculator.js file.
Develop just the UI of the calculator
Use functional components
Imported it in your top-level component App.js
Test it in the browser

### CODE REVIEW

File: src/components/pages/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      <button type="button" className="decimal">
+        .
+      </button>
+      <button type="button" className="equal orange">
+        =
+      </button>
+    </div>
+  </section>
+);
+
```
Comment (TedLivist) [inline src/components/pages/Calculator.js]: - [ ] Your calculator component should have at least one child component. Kindly create a new component in a new file and import it here.

File: src/components/pages/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      </button>
+      <button type="button" className="plmi">
+        +/-
+      </button>
+      <button type="button" className="percent">
+        %
+      </button>
+      <button type="button" className="division orange">
+        /
+      </button>
```
Comment (TedLivist) [inline src/components/pages/Calculator.js]: - [ ] This symbol should be &div;, not /, based on the design. If you do not have the sign on your keyboard consider using HTML entity code for it.

File: src/reportWebVitals.js [hunk 3]
```diff
@@ -1,15 +0,0 @@
-const reportWebVitals = (onPerfEntry) => {
```
Comment (TedLivist) [inline src/reportWebVitals.js:1]: - [ ] You have done well removing this file. However, you still need to remove these marked files in the `public` folder.

  iduw

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @mosidrum,

Good job so far!
✔️ Use of functional component
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @mosidrum, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**